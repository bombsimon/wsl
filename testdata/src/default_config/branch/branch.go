package testpkg

import "fmt"

func fn1() {
	for range []int{} {
		if true {
			fmt.Println("")

			continue
		}

		if true {
			fmt.Println("")
			continue
		}

		if true {
			fmt.Println("")
			break
		}

		if false {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			continue
		}

		if false {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			break
		}
	}
}

func fn2() {
	for range []int{} {
		if true {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")
			continue // want "missing whitespace decreases readability"
		}

		if true {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")
			break // want "missing whitespace decreases readability"
		}
	}
}
