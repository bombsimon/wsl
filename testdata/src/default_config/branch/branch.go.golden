package testpkg

import "fmt"

func fn1() {
	for range []int{} {
		if true {
			fmt.Println("")

			continue
		}

		if true {
			fmt.Println("")
			break
		}

		if true {
			fmt.Println("")
			continue
		}

		if true {
			fmt.Println("")
			fallthrough
		}

		if true {
			fmt.Println("")
			goto START
		}

		if false {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			break
		}

		if false {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			continue
		}

		if false {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			fallthrough
		}

		if false {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			goto START
		}
	}
}

func fn2() {
	for range []int{} {
		if true {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			break // want `missing whitespace above this line \(too many lines above branch\)`
		}

		if true {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			continue // want `missing whitespace above this line \(too many lines above branch\)`
		}

		if true {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			fallthrough // want `missing whitespace above this line \(too many lines above branch\)`
		}

		if true {
			fmt.Println("")
			fmt.Println("")
			fmt.Println("")

			goto START // want `missing whitespace above this line \(too many lines above branch\)`
		}
	}
}
